presentation/common/src/presentation-common/hierarchy/Key.ts

```diff
@@ -274,3 +278,23 @@ export interface LabelGroupingNodeKeyJSON extends GroupingNodeKeyJSON {
  * @public
  */
 export type NodeKeyJSON = BaseNodeKeyJSON | ECInstancesNodeKeyJSON | ECClassGroupingNodeKeyJSON | ECPropertyGroupingNodeKeyJSON | LabelGroupingNodeKeyJSON;
+
+/**
+ * Data structure that describes an instance keys select query
+ * @beta
+ */
+export interface InstanceKeysSelectQuery {
```

Comment:
While I agree that at the moment instance keys select query is the only type of query we have, I don't see any reason the interface should be named under this use case. Query string + bindings that are stored in this object can be used for any query, thus the interface should have a more generic name.

Reply:
Renamed to PresentationQuery